tensorflow_probability/python/experimental/stats/sample_stats_test.py

```diff
@@ -0,0 +1,437 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Tests for Sample Stats Ops."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# Dependency imports
+
+from absl.testing import parameterized
+
+import numpy as np
+
+import tensorflow.compat.v1 as tf1
+import tensorflow.compat.v2 as tf
+import tensorflow_probability as tfp
+from tensorflow_probability.python.internal import test_util
+
+@test_util.test_all_tf_execution_regimes
+class RunningStatsTest(test_util.TestCase):
+
+  def test_zero_running_variance(self):
+    deterministic_samples = [0., 0., 0., 0.]
+    running_var = tfp.experimental.stats.RunningVariance()
+    state = running_var.initialize()
+    for sample in deterministic_samples:
+      state = running_var.update(state, sample)
+    final_mean, final_var = self.evaluate([
+        state.mean,
+        running_var.finalize(state)])
+    self.assertEqual(final_mean, 0.)
+    self.assertEqual(final_var, 0.)
+
+  @parameterized.parameters(0, 1)
+  def test_running_variance(self, ddof):
+    rng = np.random.RandomState(seed=test_util.test_seed())
```

Comment:
This won't work in general. RandomState requires 32 bit integers as seeds, while test_seed returns something much larger. If you insist upon using test_seed like this, you should use test_util.test_np_rng() instead.